shoppingfluxexport.php

```diff
@@ -1594,14 +1594,14 @@ public function hookbackOfficeTop($no_cron = true, $forcedOrder = false)
         
                             if (! $forcedOrder && isset($orderExists['id_message']) && isset($orderExists['id_order'])) {
                                 SfLogger::getInstance()->log(SF_LOG_ORDERS, 'Order allready exists (id = '.$order->IdOrder.'): notifying ShoppingFlux', $doEchoLog);
-                                $this->_validOrders((string)$order->IdOrder, (string)$order->Marketplace, (int)$orderExists['id_order']);
+                                $this->_validOrders((string)$order->IdOrder, (string)$order->Marketplace, (int)$orderExists['id_order'], false, $currentToken['token']);
                                 continue;
                             }
         
                             $check = $this->checkData($order);
                             if ($check !== true) {
                                 SfLogger::getInstance()->log(SF_LOG_ORDERS, 'Check data incorrect - '.$check, $doEchoLog);
-                                $this->_validOrders((string)$order->IdOrder, (string)$order->Marketplace, false, $check);
+                                $this->_validOrders((string)$order->IdOrder, (string)$order->Marketplace, false, $check, $currentToken['token']);
```

Comment:
y'a pas un paramètre $check en trop là ?

Reply:
Ce commit n'a pas changé le comportement du paramètre "erreur".
`$check` correspond au message d'erreur retourné par `$this->checkData($order);`.
C'est le quatrième paramètre de `_validOrders()`, ce devrait donc être bon 👍